### PR TITLE
update elastic-expansion

### PR DESCRIPTION
**Pull request by wangtianyi2004**
## Versions 

- [ ] dev
- [ ] 3.0
- [ ] 2.1
- [ ] 2.0

## Languages

- [ ] Chinese
- [ ] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: docs/admin-manual/cluster-management/elastic-expansion.md [hunk 1]
```diff
@@ -24,131 +24,123 @@ KIND, either express or implied.  See the License for the
...
-`SHOW PROC '/frontends';`
+Typically, each FE node can handle the load operations of 10-20 BE nodes. A configuration of 3 FE nodes is sufficient to meet the requirements of most business scenarios.
 
-To view the current FE node situation.
 
-You can also view the FE node through the front-end page connection: ``http://fe_hostname:fe_http_port/frontend`` or ```http://fe_hostname:fe_http_port/system?Path=//frontends```.
+### Scale the FE out
 
-All of the above methods require Doris's root user rights.
+::: info Note:
```
Comment (KassieZ) [inline docs/admin-manual/cluster-management/elastic-expansion.md:44]: delete blank between ::: and info

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md [hunk 2]
```diff
@@ -27,144 +27,117 @@ under the License.
 
 
 
-Doris 可以很方便的扩容和缩容 FE、BE、Broker 实例。
+Doris 支持在线弹性扩容，过动态添加或移除节点，用户无需中断服务即可满足业务增长需求或降低空闲资源的浪费。扩缩容 BE 节点时，不影响集群可用性，但会涉及到数据搬迁，建议在业务闲事进行扩缩容操作。
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md:30]: 通过动态添加XXX，少了一个通字

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md [hunk 3]
```diff
@@ -27,144 +27,117 @@ under the License.
...
 
-## FE 扩容和缩容
+## 扩缩容 FE 集群
 
-可以通过将 FE 扩容至 3 个以上节点来实现 FE 的高可用。
+Doris 的 FE 节点分为以下三种角色，每一个 FE 节点都有全量的元数据信息：
 
-用户可以通过 MySQL 客户端登陆 Master FE。通过：
+* Master 节点：负责元数据的读写操作，当 Master 的元数据发生变化，会通过 BDB JE 协议同步到非 Master 几点中，统一集群中只能有一个 Master FE 节点；
```
Comment (KassieZ) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md:36]: 统一集群中 ---》 同一集群中

File: docs/admin-manual/cluster-management/elastic-expansion.md (hunk 1 above)
Comment (wangtianyi2004) [reply docs/admin-manual/cluster-management/elastic-expansion.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md (hunk 2 above)
Comment (wangtianyi2004) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md]: done

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md (hunk 3 above)
Comment (wangtianyi2004) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/admin-manual/cluster-management/elastic-expansion.md]: done

**Review by JNSimba — APPROVED** [review]

**Pull request merged**